### PR TITLE
convert objc_glue_stubs.c to D

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Comment by WalterBright** [comment]
The travis-ci failure is a build timeout.

File: src/objc_glue_stubs.d [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-#include <assert.h>
-#include <stdio.h>
+import core.stdc.stdio : printf;
 
-class FuncDeclaration;
-class Type;
-class TypeFunction;
-struct elem;
+import ddmd.func;
+import ddmd.mtype;
```
Comment (ZombineDev) [inline src/objc_glue_stubs.d]: Can you turn those into selective imports? Selective imports make it easier to know what comes from where when reading other people's code.

**Review by PetarKirov — APPROVED** [review]
Looks good, apart from one small nit.

**Comment by ZombineDev** [comment]
Yes travis timeout on osx. Why not restart it?
Should look something like that:
Restart Travis build

File: src/objc_glue_stubs.d (hunk 1 above)
Comment (WalterBright) [inline src/objc_glue_stubs.d]: All I'm doing with this is translating to D, I'm not fixing it, refactoring it, etc. It's just a rote translation. Improvements come later.

**Comment by yebblies** [comment]
Needs rebase.

**Pull request merged**